src/cpp/session/modules/SessionConsole.R

```diff
@@ -12,50 +12,3 @@
 # AGPL (http://www.gnu.org/licenses/agpl-3.0.txt) for more details.
 #
 #
-
```

Comment:
This was the only function in this file, so if you're removing it I would just delete the whole file and remove the reference in `src/cpp/session/modules/SessionConsole.cpp:256`

Reply:
✅ thanks